src/decider/agent/copyrightDeactivationClutterRemovalScript.py

```diff
@@ -93,13 +93,7 @@ def copyrightDeactivationMain(df, clutter_flag):
     pos_dict = {}
     full_table_pos = {"Entity": [], "POS_TAG": []}
 
-    for token in doc:
-      if not token.is_punct | token.is_space:
-        pos_dict[token.text] = token.pos_
-
-    for key in pos_dict:
-      full_table_pos["Entity"].append(key)
-      full_table_pos["POS_TAG"].append(pos_dict[key])
+    full_table_pos["Entity"], full_table_pos["POS_TAG"] = zip(*((token.text, token.pos_) for token in doc if not token.is_punct | token.is_space))
```

Comment:
This looks good!
Can you change the doc ==> doc2 ??

I think that is a small bug already there, Ideally it should be the doc2 for which we take POS_TAG and Entity.
@legendarykamal

Reply:
Thank you for the review , sounds good . Made the change 👍